stylesheets/stylesheet.css

```diff
@@ -616,4 +616,16 @@ footer p {
     h4.card-title {
         font-size: 0.7em;
     }
+}
+img:hover{
```

Comment:
there is a big problem!!! All the images are under same div so it is not possible to name it like that.
If i add hover like .div > .div > img it will then also work like the same :?